modules/customize-certificates-add-service-serving-mutating-webhook.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-mutating-webhook_{context}"]
+= Add the service CA bundle to a MutatingWebhookConfiguration
+
+You can annotate a MutatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+
+.Procedure
+
+. Annotate the MutatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true`.
++
+----
+$ oc annotate mutatingwebhoookconfigurations <mutatingwebhook-name> \//<1>
+     service.beta.openshift.io/inject-cabundle=true
+----
+<1> Replace `<mutatingwebhook-name>` with the name of the MutatingWebhookConfiguration to annotate:
```

Comment:
s/annotate:/annotate./
The colon was throwing me off. It looks better with a period (as you do in the following module).

Reply:
Nice catch, I changed a period to a colon in the wrong spot.